src/impl/icetransport.cpp

```diff
@@ -154,6 +133,39 @@ IceTransport::IceTransport(const Configuration &config, candidate_callback candi
 	mAgent = decltype(mAgent)(juice_create(&jconfig), juice_destroy);
 	if (!mAgent)
 		throw std::runtime_error("Failed to create the ICE agent");
+
+	// Add TURN servers
+	for (auto &server : servers) {
+		if (mTURNServersAdded++ >= MAX_TURN_SERVERS_COUNT) {
+			break;
+		}
+
+		addIceServer(server, false);
+	}
+}
+
+void IceTransport::addIceServer(IceServer server, bool warnOnSTUN) {
```

Comment:
I'm not a fan of the boolean. Why not always logging the warning and filtering out STUN servers in the constructor for loop?

Reply:
Done! Not needed anymore because we filter.